templates/bartholomew_sidebar.hbs

```diff
@@ -1,16 +1,28 @@
 <div class="menu-wrap is-narrow is-fixed-desktop" id="docsMenu">
     <aside class="menu">
         <p class="menu-label">
-            Using Bartholomew
+            Get Started
         </p>
         <ul class="menu-list">
             <li><a {{#if (active_content request.spin-path-info "/bartholomew/index" )}} class="active" {{/if}} href="{{site.info.base_url}}/bartholomew/">Introduction</a></li>
             <li><a {{#if (active_content request.spin-path-info "/bartholomew/quickstart" )}} class="active" {{/if}} href="{{site.info.base_url}}/bartholomew/quickstart">Quickstart</a></li>
-            <li><a {{#if (active_content request.spin-path-info "/bartholomew/templates" )}} class="active" {{/if}} href="{{site.info.base_url}}/bartholomew/templates">Templates</a></li>
             <li><a {{#if (active_content request.spin-path-info "/bartholomew/configuration" )}} class="active" {{/if}} href="{{site.info.base_url}}/bartholomew/configuration">Configuration</a></li>
+        </ul>
+
+        <p class="menu-label">
+            Tutorials
```

Comment:
To me, the "Writing Markdown" page doesn't feel like a tutorial.  But if you adopt the suggestion of "Sites" or "Building Sites" then it would fit nicely under there I think...?

Reply:
Went with "Building Sites", makes sense. Thank you!

Reply:
Latest commit (4) has all changes synced; and ready for review.